.cirrus.yml

```diff
@@ -4,14 +4,23 @@ env:
   CIRRUS_VAULT_AUTH_PATH: jwt-cirrusci
   CIRRUS_VAULT_ROLE: cirrusci-${CIRRUS_REPO_OWNER}-${CIRRUS_REPO_NAME}
 
-  DOCKER_USERNAME: VAULT[development/kv/data/docker/sonardockerrw data.username]
-  DOCKER_PASSWORD: VAULT[development/kv/data/docker/sonardockerrw data.access_token_rwd]
+  # Can push on repox-sonarsource-docker-builds.jfrog.io
+  ARTIFACTORY_DEPLOY_USERNAME: VAULT[development/artifactory/token/${CIRRUS_REPO_OWNER}-${CIRRUS_REPO_NAME}-docker-release username]
+  ARTIFACTORY_DEPLOY_PASSWORD: VAULT[development/artifactory/token/${CIRRUS_REPO_OWNER}-${CIRRUS_REPO_NAME}-docker-release access_token]
 
-  # Mend scan global configuration  
+  # Can pull from repox-sonarsource-docker-builds.jfrog.io
+  ARTIFACTORY_USERNAME: VAULT[development/artifactory/token/${CIRRUS_REPO_OWNER}-${CIRRUS_REPO_NAME}-docker-release username]
+  ARTIFACTORY_PASSWORD: VAULT[development/artifactory/token/${CIRRUS_REPO_OWNER}-${CIRRUS_REPO_NAME}-docker-release access_token]
+
+  # Mend scan global configuration
   MEND_API_KEY: VAULT[development/kv/data/mend data.apikey]
 
   # Staging image configuration
-  STAGING_IMAGE_NAME: sonarsource/sonarqube
+  IMAGE_VERSION: 10.1.0
```

Comment:
If I understand it well, we are always static on the image version, or nothing is broken if we are not. Why do we need this? We could just tag the staging image with the build number. The same applies to the scanner CLI version we should avoid double maintaining.
For staging images, we may not need to be so specific regarding the possible release version?

I am just afraid of more human errors and manual updates, which we would save if we didn't need these extra parameters.
WDYT?

Reply:
After our discussion, I reverted to only use the build number.